trailblazer/cli/utils/user_helper.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+from trailblazer.store.models import User
+
+LOG = logging.getLogger(__name__)
+
+
+def is_existing_user(user: User, email: str) -> bool:
+    """Check if supplied user exist in db."""
+    if not user:
+        LOG.error(f"User with email {email} not found")
+        return False
+    return True
```

Comment:
ah, I now understand why there was no LOG when checking user. Maybe we can add the other check:
```Python 
is_archived
```
here as well

Reply:
Fixed